pallets/accounts/src/lib.rs

```diff
@@ -140,8 +140,8 @@ pub struct EthLockUpdate<TPublic> {
 	/// Otherwise, if the event was `Unlock`, it must equal the ***total amount* of FRAG token that was previously locked** on the **FRAG Token Smart Contract**
 	pub amount: U256,
 	/// If the event was `Lock`, it represents the lock period of the FRAG token.
-	/// Owtherwise, if the event was `Unlock`, it is zero.
-	pub locktime: U256,
+	/// If the event was `Unlock`, it is 999.
+	pub lock_period: U256,
```

Comment:
U(256) is 4 bytes [here](https://github.com/paritytech/parity-common/blob/baebea7c3c0f90d825b7024a5ffe97870f819317/primitive-types/src/lib.rs#L38).
But yes I can even use a `u8` here since the range is `[0 - 4]`.